DEVELOPMENT_ENVIRONMENT.md

```diff
@@ -29,45 +48,38 @@
 	- If you've cloned the repository from the command line or a Git client, you obviously only need to perform the last step
 - Install [Payara Tools](https://marketplace.eclipse.org/content/payara-tools)
 - Install the [Vaadin Plugin for Eclipse](https://marketplace.eclipse.org/content/vaadin-plugin-eclipse) (no need to install the commercial UI designer)
-- Add a Payara server to Eclipse and enter the credentials you specified when setting up the server
+- Add a Payara server to Eclipse 
+    - In the **Servers**-Tab create a new server and select **Payara/Payara**
+    - Add a **Server runtime environment** 
+        - **Payara location** is `sormas-cargoserver/target/cargo/installs/payara-5.194/payara5`
+        - **Java location** has to point to a **JDK 8** home path!
+    - Set **Domain path** to `sormas-cargoserver/target/cargo/configurations/payara/sormas`
+    - **Admin password** is `adminadmin` and go to the next page
+    - Add the artifacts `sormas-ear`, `sormas-rest` and `sormas-ui` to **Configured** and **finish** the setup
+    - In the Server's **Launch configuration** add the serverlibs to the classpath: `sormas-base/dependencies/target/sormas-serverlibs-serverlibsjar.jar`
 - Configure automatic code formatting ("Window -> Preferences"):
     - Go to "Java -> Code Style -> Formatter", import ``sormas-base/java-formatter-profile.xml`` and apply.
     - Go to "Java -> Code Style -> Organize Imports", import ``sormas-base/java-importorder-profile.importorder``, "Number of imports needed for .*" = ``99``, "Number of static imports needed for .*" = ``99``, "Do not create import for types starting with a lowercase letter" = ``checked`` and apply.
     - Go to "Java -> Editor -> Save Actions", activate "Perform the selected actions on save", "Format source code" with "Format all lines", "Organize imports" and apply.
 
-### Additional Steps
```

Comment:
The deploy serverlibs part is still needed for production like dev environments, so I'd suggest to keep this whole section and to make clear that it can be skipped when using cargo.